README.md

```diff
@@ -50,9 +50,9 @@ The `docker_images` field should contain a list of the names of the images your
 ### Docker
 Your project will need to be running in Docker containers, which only require a Dockerfile and a build script. It may be beneficial to get a basic understanding of Docker from the [Official Get Started Documentation](https://docs.docker.com/get-started/). 
 
-This is the [Dockerfile](app/Dockerfile). It starts from a base image of `alpine:3.6` and installs everything in `requirements.txt`, copies over the necessary code, and adds an entrypoint, which is the command that runs when the container starts up. The comments in the code should explain each command.
```

Comment:
most of the changes here are due to the fact that the underlying yaml files are copy-pasted here (for ease of explanation)